lib/order.rb

```diff
@@ -5,18 +5,18 @@ class Order
 
   attr_reader :selection, :items, :total
 
-  def initialize
+  def initialize(items = Menu.new.items)
     @selection = []
-    @items = Menu.new.items
+    @items = items
     @total = 0
   end
 
   def view_menu
```

Comment:
Have implemented suggestion to move this to initialize, but have a question:

Which is better? The implemented initialize or the one below?

  ```
def initialize(items = Menu.new)
     @selection = []
     @items = items.items
     @total = 0
  end
```
  
  Both seem to get the same results... The first reads more clearly but makes for a longer argument